app/controllers/houses_controller.rb

```diff
@@ -61,7 +61,7 @@ def current_parties
 
     @house, @parties = data.filter('http://id.ukpds.org/schema/House', 'http://id.ukpds.org/schema/Party')
     @house = @house.first
-    @parties = @parties.sort_by(:name)
+    @parties = @parties.sort_by(:member_count).reverse
```

Comment:
The sort method within parliament-ruby only sorts one way - as it can take multiple parameters, I wasn't sure how to also allow for different sorting orders.  If you have time, perhaps you could take a look at it in parliament-ruby and make some suggestions?